release/lib/git-helpers.ts

```diff
@@ -203,6 +210,34 @@ const extractDataFromCommit = (sha: string): Promise<Commit> => {
     return commitInfoPromise;
 };
 
+const createOctokitFromToken = (token: string) => {
+    const options = Object.assign({}, { auth: `token ${token}` }, octokitOptions);
+
+    const kit = new Client(options);
+
+    return kit;
+};
+
+const createOctokitFromUserPass = (auth: GitHubAuth) => {
+    if ('token' in auth) {
+        return new Client();
```

Comment:
shouldn't this be `return createOctokitFromToken(auth.token)` ?

Reply:
That code should never be hit, it was there so TypeScript doesn't complain about possible missing properties and ESLint about not having consistent returns. I'll add it for completeness with a comment.